### DISCUSSION TITLE
Any updates coming? Or migration guide to use Fullstackhero?

### DISCUSSION THREAD

**Discussion by dotnetshadow** [fullstackhero/blazor-starter-kit#431] [Q&A]
Hi there,

Firstly a great effort with both BH and FSH, congrats.

I know you might be extremely busy, just wondering what's the roadmap for BH? Will anymore updates be coming or can we assume to start moving over to FSH even though they are slightly different projects. 

Perhaps a migration guide could help people move over if at all possible? Conversely, it would be nice to seem updates that have been made in FSH that could be used in BH.

**Comment by punkouter26** [comment]
came here for same question.. It seems better to have everything in one solution for me? Is there something I am missing ?  
So that is what I am using. It is two years old but everything still seems to work
